frontend/src/pages/operation_show/layout/toolbar/index.tsx

```diff
@@ -45,6 +47,8 @@ export const Toolbar = (props: {
       ]), [props.operationSlug]
     ))
 
+  const userCanExport = process.env.ENABLE_EVIDENCE_EXPORT && props.userCanExportData
```

Comment:
allows user to export if exporting is enabled AND if they are an admin (system, operation, or part of a group with admin privileges)